src/saveload/afterload.cpp

```diff
@@ -1045,18 +1061,18 @@ bool AfterLoadGame()
 							break;
 						case ROAD_TILE_DEPOT:    break;
 					}
-					SetRoadTypes(t, ROADTYPES_ROAD);
+					SB(_me[t].m7, 6, 2, ROADTYPES_ROAD);
```

Comment:
helper function for this?

Reply:
No, this is old savegame conversion code. At this point it is converting a pre-tram game to post-tram format, then later on it will convert this pre-nrt format to post-nrt format.